test/integration/targets/sqs_queue/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+---
+
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- block:
+  - name: Test creating SQS queue
+    sqs_queue:
+      name: "{{ resource_prefix }}{{ 1000 | random }}"
+      <<: *aws_connection_info
+    register: create_result
+  - name: Assert SQS queue created
+    assert:
+      that:
+        - create_result.changed
+        - create_result.region == "{{ aws_region }}"
+  - name: Test deleting SQS queue
+    sqs_queue:
+      name: "{{ create_result.name }}"
+      state: absent
+      <<: *aws_connection_info
+    register: delete_result
+    retries: 3
+    delay: 3
+    until: delete_result.changed
+  - name: Assert SQS queue deleted
+    assert:
+      that:
+        - delete_result.changed
+
+- name: Test queue features
+  block:
+    - name: Test create queue with attributes
+      sqs_queue:
+        name: "{{ resource_prefix }}{{ 1000 | random }}"
+        default_visibility_timeout: 900
+        delivery_delay: 900
+        maximum_message_size: 9009
+        message_retention_period: 900
+        receive_message_wait_time: 10
+        policy:
+          Version: "2012-10-17"
+          Statement:
+            Effect: Allow
+            Action: "*"
+        <<: *aws_connection_info
+      register: create_result
+    - name: Assert queue created with configuration
+      assert:
+        that:
+          - create_result.changed
+          - create_result.default_visibility_timeout == "900"
```

Comment:
Are strings, not integers, really required here and below? They're ints in docs: https://docs.ansible.com/ansible/2.7/modules/sqs_queue_module.html#sqs-queue-module
I'm asking because these tests are failing for #50686.